backend/apps/predictions/github_service.py

```
@@ -0,0 +1,71 @@
+                    data = res.json()
+                    return {
+                        "pr_number": data.get("number"),
+                        "title": data.get("title", ""),
+                        "author": data.get("user", {}).get("login", "unknown"),
+                        "additions": data.get("additions", 0),
+                        "deletions": data.get("deletions", 0),
+                        "changed_files": data.get("changed_files", 0),
+                        "requested_reviewers": [r.get("login") for r in data.get("requested_reviewers", [])],
+                    }
+            except Exception:
+                pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)